packages/app/src/debug/DebugRunNavigationRow.vue

```diff
@@ -0,0 +1,101 @@
+<template>
+  <li
+    class="mr-12px ml-6px "
+    :data-cy="isCurrentRun ? 'current-run' : 'run'"
+  >
+    <component
+      :is="isCurrentRun ? 'div': 'button'"
+      :aria-label="t('debugPage.switchToRun', {runNumber: gql.runNumber})"
```

Comment:
This aria label has some problems. I think there needs to be a `computed` for the `runNumber` to populate, as it's probably just evaluated once on render, meaning we have an empty run number in the label.

But when this renders a button, the presence of `aria-label` prevents the internal content from being exposed to screen readers, the label becomes the accessible name. When it's a div and not interactive this doesn't seem to be a problem. [Loom](https://www.loom.com/share/5cd609f2ff59433e8c9b8795cc1ba957?highlightComment=11609758&t=939).

Reply:
A click hander on a div is fine if there is a button inside that can emit a click and be the focus target for keyboard users, even if the visual style makes the whole row look selected.

Reply:
I follow all of your feedback. I will create a follow up issue for this.  I can see where we also make use of the HeadlessUI radio group component to get better keyboard navigation in that page you mention in the loom.